doc/Security.xml

```diff
@@ -1280,7 +1280,12 @@ Added certificate-based client authentication</revremark>
           <section>
             <title>UploadKeyPairInPKCS8</title>
             <para>This operation uploads a key pair in a PKCS#8 data structure as specified in [RFC 5958, RFC 5959]. </para>
-            <para>If an encryption passphrase ID is supplied in the request, the device shall assume that the KeyPair parameter contains an EncryptedPrivateKeyInfo ASN.1 structure that is encrypted under the passphrase in the keystore that corresponds to the supplied ID, where the EncryptedPrivateKeyInfo structure contains both the private key and the corresponding public key. If no encryption passphrase ID is supplied, the device shall assume that the KeyPair parameter contains a OneAsymmetricKey ASN.1 structure which contains both the private key and the corresponding public key. If a passphrase is supplied, the device shall ignore an eventually supplied passphrase ID and assume that the KeyPair parameter contains an EncryptedPrivateKeyInfo ASN.1 structure that is encrypted under the supplied passphrase, where the EncryptedPrivateKeyInfo structure contains both the private key and the corresponding public key.</para>
+            <para>If a passphrase is either directly provided or as ID reference to a previously
+              uploaded passphrase, the device shall assume that the KeyPair parameter contains an
+              EncryptedPrivateKeyInfo ASN.1 structure that is encrypted with the given passphrase.
+              In case neither a passphrase nor a passphrase ID is provided the device shall assume
+              that the KeyPair parameter contains a OneAsymmetricKey ASN.1 structure which contains
```

Comment:
I agree with the update you have made and will approve these changes but the sentence would read better if you removed the 'a' in front of "OneAsymmetricKey" to read;

`.. that the KeyPair parameter contains OneAsymmetricKey ASN.1 structure ..`